build.gradle

```diff
@@ -56,6 +58,45 @@ dependencyCheck {
   suppressionFile = 'config/dependency-check/dependency-check-suppressions.xml'
 }
 
+publishing {
+  publications {
+    Main(MavenPublication) {
+      from components.java
+      artifact sourcesJar
+      artifact shadowJar
+      groupId project.group
+      artifactId rootProject.name
+      version project.version
+    }
+  }
+}
+
+bintray {
+  user = System.getenv('BINTRAY_USER')
+  key = System.getenv('BINTRAY_KEY')
+  publications = ['Main']
+  publish = true
+  pkg {
+    repo = 'hmcts-maven'
```

Comment:
What's this?

Reply:
Its the bintray repo for the hmcts organisation (https://bintray.com/hmcts/hmcts-maven)